for_users.md

```diff
@@ -1,74 +1,49 @@
 # For Users
 
-[Jupyter Notebook](https://jupyter.org/) is a wonderful project widely used by scientists.
+For first insights and without any installation, try the OpenFisca tutorial on Binder.
 
-Basically it is a web interface running in your browser, in which you write code and it displays results.
+> Binder is a [free](http://docs.mybinder.org/faq) service which allows running [Jupyter](https://jupyter.org/) notebooks.
+> Basically it is a web interface running in your browser, in which you write code and it displays results.
 
-A easy way to run OpenFisca is then to use this interface. There are several ways to use Jupyter Notebook:
+Just click on this link: http://mybinder.org/repo/openfisca/tutorial – it might take a couple of minutes to initialize. It yields an environment with the latest version of OpenFisca-France.
```

Comment:
Merge with the first paragraph, make `OpenFisca tutorial` a link to the binder. Rephrase “yields” → “offers”.